client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReviewListEntry from './ReviewListEntry.jsx';
+
+class ReviewList extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render(props) {
```

Comment:
Also, is there a reason you use makeReviews to make an array of ReviewListEntries, rather than using map on this.props.reviews? If you use map instead, this component wouldn't need to be a class.

Reply:
Refactored to a functional component instead. Should look much cleaner now, thanks :)